core/domain/exp_domain_test.py

```diff
@@ -5158,23 +5179,34 @@ class HTMLMigrationUnitTests(test_utils.GenericTestBase):
 
     def test_load_from_v26_textangular(self):
         """Test direct loading from a v26 yaml file."""
-        with self.swap(
+        mock_get_filename_with_dimensions_context = self.swap(
             html_validation_service, 'get_filename_with_dimensions',
-            mock_get_filename_with_dimensions):
-
-            exploration = exp_domain.Exploration.from_yaml(
-                'eid', self.YAML_CONTENT_V26_TEXTANGULAR)
+            mock_get_filename_with_dimensions)
+        mock_validate_rte_format_context = self.swap(
+            html_validation_service, 'validate_rte_format',
+            mock_validate_rte_format)
+
+        with mock_get_filename_with_dimensions_context:
```

Comment:
Using them in one line requires to split them by using brackets since the line gets too long and adding a bracket raises error since with cannot find **__exit__**